source/elements/oneMKL/source/domains/spblas/data_types/set_matrix_properties.rst

```diff
@@ -0,0 +1,107 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_set_matrix_properties:
+
+set_matrix_properties
+=====================
+
+Matrix properties
+-----------------
+
+.. container:: section
+
+   .. code:: cpp
+
+      namespace oneapi::mkl::sparse {
+
+          enum matrix_property : std::int32_t {
+             symmetric = 1 << 0,
+             sorted    = 1 << 1,
+          };
+
+      }
+
+   Matrix properties are optional and "strong" guarantees. Unlike
+   :ref:`onemkl_sparse_matrix_view`, the user must ensure that the handle's data
+   holds all the given properties. A property can be set to optimize some
+   operations.
```

Comment:
```suggestion
   Matrix properties are optional and "strong" guarantees. Unlike
   :ref:`onemkl_sparse_matrix_view`, the user must ensure that the handle's data
   holds all the given properties. A property can be set as a hint for library 
   to optimize some operations.
```

Reply:
Done in https://github.com/uxlfoundation/oneAPI-spec/pull/522/commits/7247b6d78a98ffe39c269e11bf15aaeea18076f8